src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -1520,24 +1520,24 @@ in :ref:`Section External Triggers`.
 
 
 
-.. _User Guide Expected Outputs:
-.. _expected outputs:
+.. _User Guide Required Outputs:
+.. _required outputs:
 .. _incomplete tasks:
 
-Expected Outputs
+Required Outputs
 ----------------
 
 .. versionadded:: 8.0.0
 
 :term:`Task outputs <task output>` in the :term:`graph` are either
-:term:`expected <expected output>` (the default) or  :term:`optional <optional
+:term:`required <required output>` (the default) or  :term:`optional <optional
 output>`.
 
 The scheduler expects all task outputs to be completed at runtime, unless they
```

Comment:
```suggestion
The scheduler requires all task outputs to be completed at runtime, unless they
```

Reply:
In this case I was thinking more of "the scheduler expects *required outputs* to be completed".  But fair enough.